advanced_start/more.rst

```diff
@@ -0,0 +1,159 @@
+.. _more:
+
+Productionising a Classifier as an NStack Module
+================================================
+
+So far, we have built and published a Python module with a single function on it, ``numChars``, and built a workflow which connects our function to an HTTP endpoint. This in itself isn't particularly useful, so, now that you've got the gist of how NStack works, it's time to build something more realistic!
+
+In this tutorial, we're going to create and productionise a simple classifier which uses the famous `iris dataset <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_.
+
+We're going to train our classifier to classify which species an iris is, given measurements of its sepals and petals. You can find the dataset we're using to train our model `here  <https://raw.githubusercontent.com/nstackcom/nstack-examples/master/iris/irisclassify/train.csv>`_.
+
+First, let's look at the the format of our data to see how we should approach the problem. We see that we have five fields: 
+
+================  =======================  ===========
+Field Name        Description              Type
+================  =======================  ===========
+``species``       The species of iris      Text
+
+``sepal_width``   The width of the sepal   Double
+
+``sepal_length``  The length of the sepal  Double
+
+``petal_width``   The width of the petal   Double
+
+``petal_length``  The length of the petal  Double
+================  =======================  ===========
+
+If we are trying to find the species based on the sepal and petal measurements, this means these measurements are going to be the input to our classifier module, with text being the output. This means we need to write a function in Python which takes four ``Double``\s and returns ``Text``.
+
+Creating your classifier module
+******************************
+
+To begin, let's make a new directory called ``irisclassify``, ``cd`` into it, and initialise a new module:
+
+.. code:: bash
+    
+    ~/ $ mkdir Irisclassify; cd Irisclassify
```

Comment:
If we want to follow the directory structure of the examples, this should be `Iris.Classify`